src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -188,15 +197,15 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     }));
   };
 
-  fetchSavedObjects = () => {
-    this.setState({ isSearching: true }, this.debouncedFetchObjects);
+  fetchAllSavedObjects = () => {
+    this.setState({ isSearching: true }, this.debouncedFindObjects);
   };
 
-  fetchSavedObject = (type: string, id: string) => {
-    this.setState({ isSearching: true }, () => this.debouncedFetchObject(type, id));
+  fetchSavedObjects = (objects: Array<{ type: string; id: string }>) => {
+    this.setState({ isSearching: true }, () => this.debouncedBulkGetObjects(objects));
   };
```

Comment:
I renamed these to be more accurate. Also, since I needed to add support for `bulkGet` to fetch more than one object by ID, I got rid of the old API that fetched a single object by ID.